as-vending/functions/output.go

```diff
@@ -180,20 +180,31 @@ func (vendingState *VendingState) HandleMqttDeviceReading(lc logger.LoggingClien
 // VerifyDoorAccess will take the card reader events and verify the read card id against the white list
 // If the card is valid the function will send the unlock message to the device-controller-board device service
 func (vendingState *VendingState) VerifyDoorAccess(lc logger.LoggingClient, event dtos.Event) (bool, interface{}) {
-	lc.Info("new card scanned", "workflow:", vendingState.CVWorkflowStarted, "maintenance mode:", vendingState.MaintenanceMode, "open:", vendingState.DoorOpenedDuringCVWorkflow, "closed:", vendingState.DoorClosedDuringCVWorkflow, "Inference:", vendingState.InferenceDataReceived, "door:", vendingState.DoorClosed)
-
-	if event.DeviceName == "ds-card-reader" && !vendingState.CVWorkflowStarted {
+	lc.Infof("new card scanned workflow: %t maintenance mode: %t open: %t closed: %t Inference: %t door: %t",
+		vendingState.CVWorkflowStarted,
+		vendingState.MaintenanceMode,
+		vendingState.DoorOpenedDuringCVWorkflow,
+		vendingState.DoorClosedDuringCVWorkflow,
+		vendingState.InferenceDataReceived,
+		vendingState.DoorClosed)
+	if event.DeviceName == DsCardReader && !vendingState.CVWorkflowStarted {
 		lc.Info("Verify the card reader input against the white list")
 
-		lc.Info("Card Scanned", "workflow:", vendingState.CVWorkflowStarted, "maintenance mode:", vendingState.MaintenanceMode, "open:", vendingState.DoorOpenedDuringCVWorkflow, "closed:", vendingState.DoorClosedDuringCVWorkflow, "Inference:", vendingState.InferenceDataReceived, "door:", vendingState.DoorClosed)
+		lc.Infof("Card Scanned workflow: %t maintenance mode: %t open: %t closed: %t Inference: %t door: %t",
+			vendingState.CVWorkflowStarted,
+			vendingState.MaintenanceMode,
+			vendingState.DoorOpenedDuringCVWorkflow,
+			vendingState.DoorClosedDuringCVWorkflow,
+			vendingState.InferenceDataReceived,
+			vendingState.DoorClosed)
```

Comment:
Looks like opportunity for a helper function to eliminate the repeated code.

Reply:
next PR